.gitignore

```diff
@@ -0,0 +1,3 @@
+*.swp
+*.swo
+*.xml
```

Comment:
You should add .idea to your .gitignore so that all the generated IDE files are not added (you should remove them from this PR ideally)